src/sets.jl

```diff
@@ -411,6 +458,11 @@ struct PositiveSemidefiniteConeSquare <: AbstractSymmetricMatrixSetSquare
     side_dimension::Int
 end
 
+function dual_set(s::PositiveSemidefiniteConeSquare)
+    return error("""Dual of $s is not defined because $s is not a proper cone. 
+                    For more details please see comments on https://github.com/JuliaOpt/MathOptInterface.jl/blob/master/src/Bridges/Constraint/square.jl""")
```

Comment:
This is an awkward link to include in an error message. I would just say, "see the comments in src/Bridges/Constraint/square.jl".